### DISCUSSION TITLE
Shelly 3EM : Template available ?

### DISCUSSION THREAD

**Discussion by Noschvie** [arendst/Tasmota#11909] [Q&A] [answered]
Looking for a template for Shelly 3EM. Any idea ?
Thanks!

**Comment by sfromis** [comment, marked as answer]
Someone will have to be first to create one. If a template was available, it would be in the [templates repository](https://templates.blakadder.com/).

**Reply by ascillato2** [reply]
More info at

https://github.com/arendst/Tasmota/discussions/13515
